src/bgw/job_stat.c

```diff
@@ -56,8 +56,8 @@ bgw_job_stat_scan_one(int indexid, ScanKeyData scankey[], int nkeys,
 {
 	Catalog    *catalog = catalog_get();
 	ScannerCtx	scanctx = {
-		.table = catalog->tables[BGW_JOB_STAT].id,
-		.index = CATALOG_INDEX(catalog, BGW_JOB_STAT, indexid),
+		.table = catalog_table_id(catalog, BGW_JOB_STAT),
```

Comment:
convention is `catalog_get_table_id` and `catalog_get_index`

Reply:
fixed

Reply:
fixed